cartography/intel/gcp/cloudkms.py

```diff
@@ -171,7 +172,53 @@ def get_keyring_policy_entities(kms: Resource, keyring: Dict, project_id: str) -
         else:
             raise
 
+@timeit
+def transform_keryring_policy_bindings(response_objects: List[Dict], function_id: str,project_id: str) -> List[Dict]:
+    """
+    Process the GCP kms_policy_binding objects and return a flattened list of GCP bindings with all the necessary fields
+    we need to load it into Neo4j
+    :param response_objects: The return data from get_gcp_function_policy_bindings()
+    :return: A list of GCP GCP bindings
+    """
+    binding_list = []
+    for res in response_objects:
+        res['id'] = f"projects/{project_id}/function/{function_id}/role/{res['role']}"
+        binding_list.append(res)
+    return binding_list
```

Comment:
in this method make changes for keryring instead of function

Reply:
done...sorry missed in hurry